.github/workflows/benchmarks.yml

```
@@ -48,29 +49,39 @@ jobs:
       - name: Install python deps
         run: python3 -m pip install -r ./integration-tests/scripts/baseline/requirements.txt
 
-      - name: Calculate Baseline
+      - name: Compare with the Baseline
         run: |
-          jq -s 'flatten' perf.json > integration-tests/perf-all.json
+          jq -s 'flatten' ./container-logs/*/*/perf.json > integration-tests/perf-all.json
+
+          cat integration-tests/perf-all.json
```

[nit] perhaps only cat the file in debug mode?
```suggestion
          if [[ "${RUNNER_DEBUG}" == "1" ]]; then
              cat integration-tests/perf-all.json
          fi
```